sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -756,6 +756,19 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("SPARK-32237: Hint in CTE") {
```

Comment:
Why did you put this test here? Do you think this issue in only for V2 data sources?

Reply:
Oh, it’s not only for v2. I just use a similar test for modification. I will move the test to another suite